### ISSUE TITLE
Support for artefact collections

### ISSUE THREAD

**Issue by motatoes** [ImportantCool/gogonzo#22]
For example, an artefact containing a group of photos

but we need to define the exact scope of this. can we allow a collection of docs/videos to form an artefact? what about Audio and spreadsheets?

**Comment by AustinGMackell** [comment]
This is an important question and we shouldn't try to rush to a decision without testing it out, but my instinct is to define things as widely as possible. One thing that might be relevant, and might help organise the whole process/design is to put the videos of research (and/or audio in the case of interviews) at the center, and have the files and websites accessed during that research available in full, and accessible from that page. So if there was a bunch of photos (i think galleries will definitely be needed) and a spreadsheet and a website viewed by the journalist during one 2 hour research session, they'd be available in a menu accessible from the video page. Does that help at all?

**Comment by motatoes** [comment]
Yes  it does but  putting aside the research artefact my focus is on artefacts uploaded from the website/phone app

you give an artefact a title and a description:

are you then allowed to upload any group of files regardless of their type ?

**Comment by AustinGMackell** [comment]
That seems like it could get a little messy... maybe as a temporary measure--- like you can upload them like that but need to go through and individually name the files before publication... obviously that's more work but if you're not handling/examining each file then are they really part of the research process?  This is something that needs examination. Another complicating factor is that I think having a "title" and a "filename" that are distinct from each other seems like a duplication. Perhaps if they're just organised by filename, with the folder also getting a description.... But we don't want to encourage people just dumping masses of files into their writelines without knowing what's in them.

**Comment by motatoes** [comment]
Okay cool .. but galleries are an exception to that right?

**Comment by AustinGMackell** [comment]
Even then the journo should be aware of what's in the photos.

**Comment by motatoes** [comment]
Okay cool got it.
